filebeat/docs/reference/configuration/filebeat-options.asciidoc

```diff
@@ -329,6 +341,13 @@ You can force Filebeat to close the file as soon as the file name changes by set
 `force_close_files` option to true. The default is false. Turning on this option can lead to loss of data on
 rotated files in case not all lines were read from the rotated file.
 
+[[configuration-global-options]]
+=== Filebeat Global Configuration
+
+You can specify configuration options that control Filebeat behavior at a global level.
+
+==== Options
```

Comment:
I assume we need this title here, because asciidoc does not allow leaving it out?